resty/lib/src/jaguar_resty_base.dart

```diff
@@ -495,7 +495,8 @@ class Post extends RouteBase implements RouteWithBody {
 
   Post cookies(List<ClientCookie> cookies) => super.cookies(cookies);
 
-  Post body(String body) {
+  Post body(dynamic body) {
```

Comment:
For type safety, can we keep `Post body(String body)` as it is and add `Post bytes(String body)` or `Post bodyBytes(String body)`?

Reply:
As you want here, I'm personally don't mind having dynamic here as I assert the type on the method :)

Reply:
Nice. resty can be used directly without retrofit.

Reply:
let's go for bytes :)